packages/headless/package.json

```diff
@@ -8,23 +8,148 @@
   },
   "main": "./dist/headless.js",
   "module": "./dist/headless.esm.js",
-  "browser": {
-    "./dist/headless.esm.js": "./dist/browser/headless.esm.js",
-    "./dist/headless.js": "./dist/browser/headless.js"
+  "exports": {
```

Comment:
This field is the big change in this PR. It defines more clearly what import file to use at which moment.